Packs/AWS_Bedrock/Integrations/AwsBedrock/AwsBedrock_test.py

```diff
@@ -0,0 +1,35 @@
+import botocore.session
+import json
+import io
+from botocore.response import StreamingBody
+from botocore.stub import Stubber
+
+
+def util_load_json(path):
+    with io.open(path, mode='r', encoding='utf-8') as f:
+        return json.loads(f.read())
+
+
+def test_ask_command():
+    """
+    Given:
+    - A basic question.
+    When:
+    - Calling aws-bedrock-ask command.
+    Then:
+    - Ensure that the result is correct and JSON.
+    """
+    from AwsBedrock import ask_command
+
+    bedrock = botocore.session.get_session().create_client('bedrock-runtime', region_name='us-east-1')
```

Comment:
We don't want to use in any external functions in the unit test, instead we are using mock please see [here](https://xsoar.pan.dev/docs/integrations/unit-testing#mocking) 
[You can see exmples in our repo to mocking unit test 
Packs/AWS-Route53/Integrations/AWSRoute53/AWSRoute53_test.py](https://github.com/demisto/content/blob/2fab65a0ecc88352fa7e3585e5a65554215401db/Packs/AWS-Route53/Integrations/AWSRoute53/AWSRoute53_test.py#L4)

Reply:
I just modified, let me know if it matches what you expect